TACOS-rest/src/main/java/cz/fi/muni/TACOS/rest/controlers/OrderControler.java

```diff
@@ -69,6 +69,15 @@ public void deleteOrder(@PathParam("id") Long id) {
 		return orderFacade.getAllForState(state);
 	}
 
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	@Path("/filter/notNew")
```

Comment:
Im pretty sure this path conflicts with the path above. It can not be decided if this method or the one above shoul be called for "/filter/notNew".

Reply:
I'm think, that this path haven't got conflicts with path for method getAllForState, because for this method is path for example '/filter/notNew', but for method getAllForState is path for example '/filter/state/NEW'